packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.tsx

```diff
@@ -101,24 +100,17 @@ const PositionsModalCard = observer(
 
         const contract_vanilla_el = (
             <React.Fragment>
-                <NavLink
-                    className='positions-modal-card'
-                    to={{
-                        pathname: `/contract/${contract_info.contract_id}`,
-                    }}
-                >
-                    <ContractCard.Header
-                        contract_info={contract_info}
-                        display_name={display_name}
-                        getCardLabels={getCardLabels}
-                        getContractTypeDisplay={getContractTypeDisplay}
-                        has_progress_slider={!is_mobile && has_progress_slider}
-                        is_mobile={is_mobile}
-                        is_sell_requested={is_sell_requested}
-                        onClickSell={onClickSell}
-                        server_time={server_time as moment.Moment}
-                    />
-                </NavLink>
+                <ContractCard.Header
```

Comment:
In the end, all these children are wrapped with `<BinaryLink />`, so there is no need in extra link. I also checked if user is able to navigate to Contract Details page by clicking on Vanilla contract Card (works as expected) and add this point to checklist